### ISSUE TITLE
Can't disable scrollPageIntoView while changing pages in Safari

### ISSUE THREAD

**Issue by fncamm** [VadimDez/ng2-pdf-viewer#418]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

On Safari when I change a page by clicking on buttons it automatically scrolls to the start of a pdf page, so the top edge of browser's window matches top edge of the pdf page.
It doesn't work in Chrome, but if I remove `<!DOCTYPE html>` from the index.html then it works in Chrome.

I'm using version 5.2.3.
Here is code I have.

    <pdf-viewer #pdfViewer *pageSelector="let currentPage = page, let showPages = onLoad" class="viewer"
              [src]="src"
              [original-size]="false"
              [fit-to-page]="false"
              [page]="currentPage"
              [zoom]="0.8"
              [render-text]="true"
              [show-all]="false"
              [autoresize]="true"
              (after-load-complete)="showPdf(); pdfViewer.updateSize();"
              (after-load-complete)="showPages($event, holdPage ? currentPage : 1); holdPage = false;">
          </pdf-viewer>

I tried to add `[stick-to-page]="false"` but it didn't help.

**Comment by falinsin** [comment]
Please, in my case I want not scroll on every page change. Is unconfortable is there any property where desactivate this functionality. 
Wy force this scroll on every page change

**Comment by VadimDez** [comment]
Fixed in `6.0.0` See https://github.com/VadimDez/ng2-pdf-viewer/blob/master/CHANGELOG.md#600

**Comment by CesarUCIGIT** [comment]
I have found a workaround that doesn't involve directly changing the pdfjs module. So what you need to do is to capture on your component the PdfViewerComponent. Then you can access the pdfviewer and inside there is an object call scroll.down and you can set it to false. And that's it you will no longer have scrolling when changing pages. 

**Angular Component**

 ```
@ViewChild(PdfViewerComponent, { static: false })
    private pdfComponent: PdfViewerComponent;

 initLoadCompleted(pdf: PDFDocumentProxy): void {
        this.pdfComponent.pdfViewer.scroll.down = false;
    }

```
**HTML**

```
<pdf-viewer
                  [src]="pdfData" (after-load-complete)="initLoadCompleted($event)"
                  [render-text]="true"
                  [page]="currentPage"
                  [show-all]="false">
      </pdf-viewer>
```
